blockchain/accept.go

```diff
@@ -219,23 +219,6 @@ func ticketsRevokedInBlock(bl *dcrutil.Block) []chainhash.Hash {
 	return tickets
 }
 
-// voteBitsInBlock returns a list of vote bits for the voters in this block.
-func voteBitsInBlock(bl *dcrutil.Block) []stake.VoteVersionTuple {
```

Comment:
This is already due for removal in a davec PR

Reply:
Removed in #1015.